ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -103,8 +104,15 @@ ParallelConfig determine_parallel_config(
     } else if (shard_layout == TensorMemoryLayout::BLOCK_SHARDED) {
         uint32_t start_divisor =
                 block_shard_orientation == ShardOrientation::COL_MAJOR ? compute_grid_size.x : compute_grid_size.y;
+        auto channels_per_core = std::ceil((float)input_channels / effective_tile_width);
+        if(is_non_tile_mul_width) {
+            out_nhw_ntiles = tt::round_up(batch_size * output_height * output_width, tt::constants::TILE_HEIGHT) / tt::constants::TILE_HEIGHT;
+            out_c_ntiles = tt::round_up(output_channels, effective_tile_width) / 1;
```

Comment:
This is needed since non-tile multiple height does not support Block sharding and is_out_tiled variable is used to pass status of non-tile multiple height variable.